kubeadm/v1.15.0/KubeCluster.ps1

```diff
@@ -0,0 +1,288 @@
+#
+# Copyright 2019 (c) Microsoft Corporation.
+# Licensed under the MIT license.
+#
+Param(
+    [parameter(Mandatory = $false,HelpMessage="Print the help")]
+    [switch] $help,
+    [parameter(Mandatory = $false,HelpMessage="Initialize Windows Control Plane node (unsupported)")]
+    [switch] $init,
+    [parameter(Mandatory = $false,HelpMessage="Install pre-requisites")]
+    [switch] $InstallPrerequisites,
+    [parameter(Mandatory = $false,HelpMessage="Join the windows node to the master")]
+    [switch] $join,
+    [parameter(Mandatory = $false,HelpMessage="Reset this windows node and cleanup everything")]
```

Comment:
[2]
is it possible to define these HelpMessages only once and use them in both the params and usage screen?

Reply:
Unfortunately no, we need literals here. Constant variables don't work either.